### PR TITLE
Add ability to skip bucket flush when installing SG

### PR DESCRIPTION
**Pull request by tleyden**
This was needed to retest https://github.com/couchbase/sync_gateway/issues/1475

### CODE REVIEW

File: provision/ansible/playbooks/install-sync-gateway-package.yml [hunk 1]
```diff
@@ -36,7 +36,8 @@
 
   tasks:
   - include: tasks/flush-server-buckets.yml
-
+    when: skip_bucketflush == "false"
+
```
Comment (sethrosetter) [inline provision/ansible/playbooks/install-sync-gateway-package.yml]: Should this be true?

Comment (sethrosetter) [inline provision/ansible/playbooks/install-sync-gateway-package.yml]: Does a python False get correctly converted to "false" here? Just double checking.

Comment (tleyden) [inline provision/ansible/playbooks/install-sync-gateway-package.yml]: That looks right to me .. it only flushes server buckets if *skipping* bucketflush is false.  

I agree though it's a bit confusing.  

These variations would be less confusing:

```
   tasks:
    - include: tasks/flush-server-buckets.yml		    
      unless: skip_bucketflush == "true"
```

```
   tasks:
    - include: tasks/flush-server-buckets.yml		    
      when: do_bucketflush == "true"
```

Comment (tleyden) [inline provision/ansible/playbooks/install-sync-gateway-package.yml]: > Does a python False get correctly converted to "false" here? Just double checking.

It seemed to in my testing.  To be honest I was confused on how to do this comparison .. I'll try to switch to compare against False (boolean) and see what happens

**Comment by tleyden** [comment]
Test results:

https://gist.github.com/tleyden/7ffab1941425ff286426

New commit will be pushed soon

**Comment by tleyden** [comment]
@sethrosetter should be ready to merge now.

**Pull request merged**